clusters/app.ci/prow-monitoring/mixins/_prometheus/alertmanager_default_receivers.libsonnet

```diff
@@ -1,5 +1,13 @@
 {
   alertmanagerReceivers+:: [
+    {
+      name: 'pager-duty',
+      pagerduty_configs: [
+        {
+          service_key: '${PAGERDUTY_INTEGRATION_KEY}',
```

Comment:
I guess you select Prometheus as the integration type in PagerDuty.

```
If you select Prometheus as the integration type in PagerDuty, you will need to use the Events API v1 type and set a value for the service_key property instead.
```

Reply:
Yes